e2e-tests/all-features-npm.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -e
+
+DIR=$PWD
+TMP=$(mktemp -d)
+
+cd $TMP
+
+$DIR/bin/index.js --noQuest --packageManager=npm
+
+cd fp-ts-lib
+
```

Comment:
I wonder if there's a nice way to simulate an environment that lacks yarn. Otherwise future mistakes like the one you spotted will not lead to failure.
Unfortunately both e2e tests don't run in an isolated env. As the npm test runs second. Maybe something like `npm -g uninstall yarn`?